client/src/app/videos/+video-watch/video-watch.component.html

```diff
@@ -35,7 +35,7 @@
               <h1 class="video-info-name">{{ video.name }}</h1>
 
               <div i18n class="video-info-date-views">
-                Published {{ video.publishedAt | myFromNow }} - {{ video.views | myNumberFormatter }} views
+                Published on {{ video.originalPublishedAt | date: 'dd MMMM yyyy' }} - {{ video.views | myNumberFormatter }} views
```

Comment:
Do we really want to display the `originalPublishedAt` field by default? @Nutomic @rigelk

Reply:
We had settled on some variant of https://github.com/Chocobozzz/PeerTube/issues/946#issuecomment-432253009

Also please note the date format varies: https://en.wikipedia.org/wiki/Date_format_by_country | Do we already deal with date formats by language, or is it fine to leave that aside for now (probably?).